File: Linguagem C Exercicios/Vetores/SOMA VETORES.c
#include <stdio.h>
#include <stdlib.h>

int main()
{

    int n;
    double media,soma;

    printf("Quantos numeros você vai digitar? ");
    scanf("%d",&n);

    double vet[n];

    for (int i=0;i<n;i++){
      printf("Digite um numero: ");
      scanf("%lf",&vet[i]);
    };


    printf("\nVALORES: ");

    for (int i = 0;i<n;i++){
        printf("%.1lf ",vet[i]);
    };
    printf("\n");

    soma = 0;
    media = 0;

    for (int i=0;i<n;i++){
        soma = soma + vet[i];
    };

    media = soma / n;

    printf("SOMA = %.2lf\n",soma);
    printf("MEDIA = %.2lf\n",media);


    return 0;
}

File: Linguagem Python/Vetores/SomaVetores.py
n = int(input("Quantos elementos vai ter o vetor? "))

vet: [float] = [0 for x in range(n)]

for i in range(0,n):
    vet[i] = float(input("Digite um numero: "))

print()
soma = 0
for i in range(0,n):
    soma = soma + vet[i]

media = soma / n
print(f"MEDIA DO VETOR = {media:.3f}")

print("ELEMENTOS ABAIXO DA MEDIA:")
for i in range(0,n):
    if vet[i] < media:
        print(f"{vet[i]:.1f}")